requirements.txt

```diff
@@ -1,5 +1,7 @@
 viam-sdk
-numpy
+typing-extensions
+numpy==1.24.1
```

Comment:
This is pretty strict, are you able to use a >= for this, or is really important that this version of numpy be used?